client/lib/push-notifications/service-worker.js

```diff
@@ -0,0 +1,19 @@
+/**
+ *  This file is served as-is as /service-worker.js
+ *  @TODO: Change this :)
```

Comment:
Kind of related to your TODO, but I'm concerned about having such a generically-named file (`/service-worker.js`) defined & registered within a library for a specific feature (`push-notifications`).

While I'm hesitant to set a precedent that we register a new service worker for each feature that leverages one, is it possible to make the file-name more specific to notifications here while we sure up our plans to work this into the bundler?